about.html

```diff
@@ -11,55 +11,98 @@
 
 </head>
 
-<body>
-    <div class="container about-page">
+<div class="container">
+
+  <header class="nav bar">
+
+    <img src="img/lambda-black.png" alt="logo">
+    
+    <nav>
+     <a href="#">Home</a> 
+     <a href="index.html">About</a>
+     <a href="#">Products</a> 
+     <a href="#">Blog</a> 
+     <a href="#"> Contact</a>
+    </nav>
+
+  </header>
+  <body>
```

Comment:
Make sure that all of your content is inside of your body tag - this should go above your container div